### PR TITLE
System jobs tasks grouping

### PR DESCRIPTION
**Pull request by assafchamoy**
Added new TasksGroup type
Implemented tasks group logic
seperate job data from tasks

### CODE REVIEW

File: src/graphql/resolvers/task.resolver.ts [hunk 1]
```diff
@@ -45,26 +47,92 @@ export class TaskResolver {
...
+        maxDate: null,
+      };
+
+      for (const { created, updated } of taskGroup) {
+        dates.createdDates.push(moment(created as Date));
+        dates.updatedDates.push(moment(updated as Date));
+      }
+
+      dates.minDate = moment.min(dates.createdDates as unknown as moment.Moment[]);
+      dates.maxDate = moment.min(dates.updatedDates as unknown as moment.Moment[]);
```
Comment (alebinson) [inline src/graphql/resolvers/task.resolver.ts:83]: moment.min --> moment.max

**Review by alebinson — CHANGES_REQUESTED** [review]
see comment

File: src/graphql/resolvers/task.resolver.ts (hunk 1 above)
Comment (assafchamoy) [reply src/graphql/resolvers/task.resolver.ts:83]: fixed.

**Review by alebinson — APPROVED** [review]
good

**Pull request merged**